base_layer/wallet/src/transaction_service/offline_signing/marshal_output_pair.rs

```diff
@@ -0,0 +1,71 @@
+use serde::{Deserialize, Serialize};
+use tari_core::transactions::{
+    transaction_key_manager::{error::KeyManagerServiceError, TariKeyId, TransactionKeyManagerInterface},
+    transaction_protocol::sender::OutputPair,
+};
+use tari_utilities::hex::{from_hex, Hex};
+
+use crate::transaction_service::error::TransactionServiceError;
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct MarshalOutputPair {
+    pub output_pair: OutputPair,
+    pub encrypted_kernel_nonce: String,
+    pub encrypted_sender_offset_key: Option<String>,
+    pub encrypted_output_spending_key: String,
```

Comment:
```suggestion
    pub encrypted_output_commitment_mask: String,
```
this will cause confusing, we should try avoid using spending key

Reply:
Renamed - [da78928](https://github.com/tari-project/tari/pull/7122/commits/da789281dbaff37cb1732101349269c9dcdbca86)